consensus/ethash/consensus.go

```diff
@@ -663,6 +663,9 @@ func AccumulateRewards(config *chain.Config, header *types.Header, uncles []*typ
 
 // accumulateRewards retrieves rewards for a block and applies them to the coinbase accounts for miner and uncle miners
 func accumulateRewards(config *chain.Config, state *state.IntraBlockState, header *types.Header, uncles []*types.Header) {
+	if config.IsNormalcy(header.Number.Uint64()) {
```

Comment:
This is strange to me...
I thought that we need to apply rewards in any case (in fact it would have more sense to me to do it in case the normalcy HF is enabled, and otherwise skip it), since the "vanilla" behavior in the EVM should be to calculate and distribute the rewards to the block proposer/miner. WDYT?

Reply:
The sequencer in L2 shouldn't be allowed to mine new coins out of thin air. Imagine if the L2 gas token is ETH, and a sequencer mines millions of blocks and accumulates 1k ETH as reward, who is going to pay for those ETH rewards when the sequencer is trying to bridge those ETH back to L1?

Reply:
Ok that makes sense. But in that case I thought that we should disable the rewards distribution no matter which HFs are enabled (because cdk erigon should be always run as an L2)?

Reply:
Not sure.. maybe ethash consensus was used when normalcy is disabled.

Reply:
Verified that the balance of sequencer doesn't change between blocks when normalcy is disabled.